proxyd/config.go

```diff
@@ -32,8 +32,9 @@ type ServerConfig struct {
 }
 
 type CacheConfig struct {
-	Enabled bool         `toml:"enabled"`
-	TTL     TOMLDuration `toml:"ttl"`
+	Enabled  bool         `toml:"enabled"`
+	TTL      TOMLDuration `toml:"ttl"`
+	BlockTTL TOMLDuration `toml:"block_ttl"`
```

Comment:
block_ttl could probably be changed to something live ShortLivedTTL to better reflect it can be used for multiple RPCs